G/GMP/build_tarballs.jl

```diff
@@ -35,10 +28,6 @@ fi
 
 ./configure --prefix=$prefix --build=${MACHTYPE} --host=${target} ${flags[@]}
 
-# Something is broken in the libtool that gets generated on macOS; I can't
-# figure out why, but `hardcode_action` is set to blank for CXX files.  /shrug
-sed -i -e 's&hardcode_action=$&hardcode_action=immediate&g' libtool
```

Comment:
This was added by @staticfloat in d2b4c5396cfc0cec470e5696b181d9467281fcfe but once again I don't know how exactly the issues it works around manifested; GMP 6.2.0 builds fine without it. But perhaps the issue was not about a build issue but rather about an issue with using the resulting libraries?

Reply:
I believe this was an issue while building, so if it builds without doing this, great!